external-crates/move/crates/move-vm-runtime/src/interpreter.rs

```diff
@@ -258,8 +257,9 @@ impl Interpreter {
                         .instantiate_generic_function(idx, current_frame.ty_args())
                         .map_err(|e| set_err_info!(current_frame, e))?;
                     let func = resolver.function_from_instantiation(idx);
-                    let _func_name = func.pretty_string();
-                    profile_open_frame!(gas_meter, _func_name.clone());
+                    #[cfg(feature = "gas-profiler")]
```

Comment:
I was asked to remove this by Tim because he said it was 100% redundant. I don't have strong feelings about it but that was what happened

Reply:
Maybe @tzakian can share more context -- but it doesn't seem redundant completely redundant to me -- there's a possibility that without it, you would generate a string that you don't use when the gas profiler is disabled.

I vaguely recall the discussion in this area, and I had interpreted it to be about adding a feature gate on the macro call (`profile_open_frame!`) which is indeed redundant.

Reply:
Agreed that we need the feature gate on the generation of the `func_name` here.

I think what I failed to communicate properly here was the fact that we were also feature gating the call to the `profile_open/close_frame` macros, and the additional feature gates around the call to these macros were redundant since those macros expand into feature-gated code. Apologies for any miscommunication!